rotkehlchen/db/checks.py

```diff
@@ -33,7 +33,7 @@ def replacer(match: Any) -> str:
         # else
         return s
 
-    return WHITESPACE_RE.sub(replacer, text).replace(' ', '').replace('\n', '').replace('"', "'").lower()  # noqa: E501
+    return WHITESPACE_RE.sub(replacer, text).replace(' ', '').replace('\n', '').replace('"', "'").replace('\t', '').lower()  # noqa: E501
```

Comment:
schema comparison failed during sanity checks because the parser only handled spaces and newlines characters, whereas my db gui created the tables with tab characters(which is also valid).